src/parser/statement.js

```diff
@@ -1418,7 +1418,8 @@ export default class StatementParser extends ExpressionParser {
       this.state.type.keyword === "let" ||
       this.state.type.keyword === "function" ||
       this.state.type.keyword === "class" ||
-      this.isContextual("async")
+      this.isContextual("async") ||
+      (this.hasPlugin("decorators2") && this.match(tt.at))
```

Comment:
Can we use `this.match(tt.at) && this.expectPlugin("decorators2")`?

Reply:
okay changed